PhysicsTools/PatAlgos/plugins/PATMuonProducer.cc

```diff
@@ -815,6 +836,94 @@ double PATMuonProducer::getRelMiniIsoPUCorrected(const pat::Muon& muon, float rh
   return pat::muonRelMiniIsoPUCorrected(muon.miniPFIsolation(), muon.p4(), drcut, rho);
 }
 
+
+double PATMuonProducer::puppiCombinedIsolation(const pat::Muon& muon, const pat::PackedCandidateCollection *pc)
+{
+  double dR_threshold = 0.4;
+  double dR2_threshold = dR_threshold * dR_threshold ;
+  double mix_fraction = 0.5;
+  const std::vector<pat::PackedCandidate> * candidates = pc;
```

Comment:
why is this assignment needed? Using ```pc``` directly is simpler.

Reply:
@slava77  done! #26233